kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/KindInstallKT.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.condition.EnabledIf;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * An installation test which depends on Kind/{@code kind}.
+ * This test:
+ * <ul>
+ * <li>assumes kind a kind cluster is available, so the test will be skipped if it's not.</li>
+ * <li>loads the image using {@code kind load image-archive}.</li>
+ *
+ * </ul>
+ */
+@EnabledIf("io.kroxylicious.kubernetes.operator.KindInstallKT#areToolsInstalled")
+class KindInstallKT extends AbstractInstallKT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KindInstallKT.class);
+
+    // These are normally set automatically by mvn
```

Comment:
We can drop this comment now, or change it to explain that the `operator-info.properties` file should be put on the classpath as a result of the maven build.